scripts/mean_guide

```diff
@@ -7,6 +7,7 @@ HASH="${1#"${1%???}"}"
 
 ALIGPATH="/extra/wayne1/research/drdavis/SDSS/FITS/2020/SDSS_DR12_Aligned/ByColor"
 MEANPATH="/extra/wayne1/research/drdavis/SDSS/FITS/2020/SDSS_DR12_Aligned/AllColorsSummed"
+ELPSPATH="/home/kimiay/sparc_stuff/elps_dir"
```

Comment:
Erm... this won't work anywhere other than openlab, so it's best to make this directory optional and given on the command-line.